src/fizziks.c

```diff
@@ -371,3 +371,51 @@ void DrawSoftBody(SoftBody *soft_body)
         DrawParticle(soft_body->springs[i].particles[1]);
     }
 }
+
+
+//Methods for Simple Shapes
+SoftBody CreateSquare(Particle * particles, Spring * springs, float init_x, float init_y, float width, float part_mass, float stiffness, int num_diagonals, bool flag_right_diagonal) {
```

Comment:
pointer-ს სადაც წერ გამოტოვება არ გინდა: `Particle *particle` ესე შეგიძლია დაწერო `Particle * particles`-ს მაგივრად. (Just purely for consistency, no other reason.)